spec.md

```diff
@@ -317,6 +319,40 @@ service Controller {
 
   rpc ControllerGetCapabilities (ControllerGetCapabilitiesRequest)
     returns (ControllerGetCapabilitiesResponse) {}  
+
+  // Snapshot can be used as the source to provision a new volume.
```

Comment:
I think I got comments in the previous meeting to add comments here so that these comments will be shown in the generated csi.proto:).  I can certainly move them to the relevant section below.

Reply:
Moved comments down to the relevant section below.